tests/ignite/contrib/metrics/test_fid.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pytest
+import torch
+import torchvision
+import torchvision.transforms as transforms
+
+from ignite.contrib.metrics import FID
+
+try:
+    from scipy import linalg
+except ImportError:
+    raise RuntimeError("This contrib test requires scipy to be installed.")
+
+
+def test_fid():
+
+    size = 10
+
+    y = torchvision.datasets.ImageFolder(
```

Comment:
I think we can read the image with Pillow and use torchvision to transform it to tensor without using `datasets.ImageFolder`.
```python
from pathlib import Path

import PIL
from torchvision.transforms.functional import to_tensor


@pytest.fixture()
def img_filepath():
    fp = Path(__file__).parent / "assets" / "fid_sample.jpg"
    assert fp.exists()
    yield fp.as_posix()


def test_fid(img_filepath):
    pil_img = PIL.Image.open(img_filepath)
    y = to_tensor(pil_img)
    y_pred = y + torch.rand_like(y) * 0.01

```